src/Dialog/index.stories.js

```diff
@@ -176,11 +176,6 @@ FocusManagement.parameters = {
 export default {
   title: "Components/Dialog",
   component: Dialog,
-  parameters: {
-    docs: {
-      page: DialogLayout,
-    },
-  },
```

Comment:
In a follow up, we will instead create an autodocs layout to use here. For now, it can use the default layout without any issues.